cmd/ocm/login/cmd.go

```diff
@@ -313,6 +313,10 @@ func run(cmd *cobra.Command, argv []string) error {
 		gatewayURL = fmt.Sprintf("https://%s", regValue.URL)
 	}
 
+	if overrideUrl := os.Getenv("OCM_URL"); overrideUrl != "" {
```

Comment:
I've recently created this properties file we can throw env variable in https://github.com/openshift-online/ocm-cli/blob/main/pkg/properties/properties.go

Reply:
Nice - updated :D